vignettes/sircovid-fits.Rmd

```diff
@@ -0,0 +1,553 @@
+---
+title: "Fitting and forecasting with SIRCOVID"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{sircovid-fits}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Overview of the SIRCOVID code
+
+The epidemiological model is implemented in the [`odin`](https://mrc-ide.github.io/odin/) DSL, and compiled using [`odin.dust`](https://mrc-ide.github.io/odin.dust/). The code can be found in `inst/odin/carehomes.R`.
+
+The model can be run using the [`dust`](https://mrc-ide.github.io/dust/) package interface, given all necessary parameters and initial conditions. Typically however, `sircovid` internally uses `dust` and `mcstate` to provide functions to fit parameters, and simulate the model to create forecasts or counterfactuals.
+
+# Overview of the structure of the SIRCOVID model
+
+`sircovid` implements a state-space model (also known as a compartmental model) for the transmission of COVID-19 in the UK, starting from 2020-Jan-01. The model, at its core, is an [SEIR model](https://en.wikipedia.org/wiki/Compartmental_models_in_epidemiology#The_SEIR_model). On top of this we have added age structure, hospitalisation pathways, seroprevalence testing and a parallel epidemic in carehomes.
+
+![Graphical summary of the sircovid model](../man/figures/sircovid_diagram.png "sircovid partitions"){ width = 10cm}
+
+Individuals are initially susceptible ($S$) in age groups $i$, typically in 17 five year bands up to 80+. Two additional bands are used to separately model care-home workers, and care-home residents. Individuals are infected (into $E$) with rate $\lambda^i(t)$ which is calculated from age-specific mixing of infected $I$ individuals. Individuals may be asymptomatic $I_A$, develop mild symptoms $I_M$ or influenza-like illness (ILI) $I_{\mathrm{ILI}}$. Individuals recover into $R$ - presently we do not model reinfection. Some proportion $p_H$ of those from $I_{\mathrm{ILI}}$ will develop severe symptoms, in which case they may:
+
+- Be admitted into hospital with confirmed COVID-19 with probability $(1-p_{CD})(p_{AC})$.
+- Be admitted into hospital, without confirmed COVID-19 with probability $(1-p_{CD})(1 - p_{AC})$.
+- No admission to hospital, leading to death in the community $I_{C_D}$, with probability $p_{CD}$.
+
+Of those in general hospital beds, two parallel pathways for confirmed and unconfirmed cases exist, along which individuals may:
+
+- Have their COVID-19 case confirmed ar rate $\gamma_{AC}$.
```

Comment:
A good question. It's describing the diagram rather than parameters in the code so may be slightly more stable, and there is a description below of how to extract current model parameters from the odin code.

It feels useful to attempt some description of the model (just like including the figure in the package), so would like to keep this, but have added a note that this may be out of date, and the paper/odin model are the places to look.